lib/mini_sql/postgres/prepared_connection.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module MiniSql
+  module Postgres
+    class PreparedConnection < Connection
+
+      attr_reader :unprepared
+
+      def initialize(unprepared_connection, deserializer_cache)
+        @unprepared         = unprepared_connection
+        @raw_connection     = unprepared_connection.raw_connection
+        @deserializer_cache = deserializer_cache
+        @type_map           = unprepared_connection.type_map
+        @param_encoder      = unprepared_connection.param_encoder
+
+        @prepared_cache     = PreparedCache.new(@raw_connection)
+        @param_binder       = PreparedBinds.new
+      end
+
+      def build(_)
+        raise 'Builder not may create on prepare connection, instead `::MINI_SQL.prepared.build(sql).query` use `::MINI_SQL.build(sql).prepared.query`'
```

Comment:
```suggestion
        raise 'Builder can not be called on prepared connections, instead of `::MINI_SQL.prepared.build(sql).query` use `::MINI_SQL.build(sql).prepared.query`'
```